docs/dfx-docs.css

```diff
@@ -44,3 +44,9 @@ table.mainbox
 	border-style: solid ;
 	border-spacing: 0.7em ;
 }
+
+img.centered {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
```

Comment:
Also should be noted that, due to the fact that these doc files are distributed independently of our website, they use their own CSS file.  I saw the addition you made to dfx.css so I copied it into this CSS file.

Reply:
Ack and then I made it more confusing because I mixed up which website CSS file you had edited in your commit, and yes you are correct that it was dfx-docs.css and I misread and thought it was dfx.css, hence confusing both of us.  Anyway, I have rectified it and reconciled all of the changes that you made to site/docs/dfx-docs.css into docs/dfx-docs.css. 😵